### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by jfgoog**

### CODE REVIEW

File: unit-test/app/src/main/cpp/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        $<TARGET_OBJECTS:adder>
+        native-lib.cpp)
+
+# Searches for a specified prebuilt library and stores the path as a
+# variable. Because CMake includes system libraries in the search path by
+# default, you only need to specify the name of the public NDK library
+# you want to add. CMake verifies that the library exists before
+# completing its build.
+
+find_library( # Sets the name of the path variable.
```
Comment (DanAlbert) [inline unit-test/app/src/main/cpp/CMakeLists.txt:38]: I don't know why all the existing samples insist on this pattern. `target_link_library(blah PRIVATE log)` works just fine.

File: unit-test/app/src/main/cpp/adder.h [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#ifndef UNIT_TEST_ADDER_H
```
Comment (DanAlbert) [inline unit-test/app/src/main/cpp/adder.h:1]: `#pragma once`

**Review by DanAlbert — APPROVED** [review]

File: unit-test/app/src/main/cpp/adder.h (hunk 2 above)
Comment (jfgoog) [reply unit-test/app/src/main/cpp/adder.h:1]: Done.

File: unit-test/app/src/main/cpp/CMakeLists.txt (hunk 1 above)
Comment (jfgoog) [reply unit-test/app/src/main/cpp/CMakeLists.txt:38]: This is from the Android Studio native app template. I'm going to leave it as-is.

**Pull request merged**